app/src/main/java/com/geeksville/mesh/ui/SignalInfo.kt

```diff
@@ -17,11 +17,14 @@ fun signalInfo(
     nodeInfo: NodeInfo,
     isThisNode: Boolean
 ): Boolean {
-    val text = if (isThisNode) {
-        "ChUtil %.1f%% AirUtilTX %.1f%%".format(
-            nodeInfo.deviceMetrics?.channelUtilization,
-            nodeInfo.deviceMetrics?.airUtilTx
-        )
+    var text = ""
+    text = if (isThisNode) {
+        nodeInfo.deviceMetrics?.let { metrics ->
+            "ChUtil %.1f%% AirUtilTX %.1f%%".format(
+                metrics.channelUtilization,
+                metrics.airUtilTx
+            )
+        } ?: ""
```

Comment:
I don't think this is necessary. device metrics should never be null. if it is, this makes it clear to users something is wrong.